refinery/user_files_manager/views.py

```diff
@@ -31,14 +33,19 @@ def user_files_csv(request):
     cols = settings.USER_FILES_COLUMNS.split(',')
 
     writer = csv.writer(response)
-    writer.writerow(cols)
+    writer.writerow(['url'] + cols)
+    # DOWNLOAD_URL's internal solr name not good for end-user.
 
     docs = loads(solr_response)['response']['docs']
     for doc in docs:
-        row = []
+        row = [doc.get(NodeIndex.DOWNLOAD_URL) or '']
         for col in cols:
-            row.append(doc.get(col + '_Characteristics_generic_s') or
-                       doc.get(col + '_Factor_Value_generic_s') or '')
+            possibly_unicode = (
```

Comment:
Might be good to have a clearer name than `possibly_unicode`? Though, it makes the `unidecode` call fairly clear